pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -44,6 +46,17 @@ const (
 	ResponseMsgType = PIURI + "/response"
 	// AckMsgType defines the did-exchange ack message type.
 	AckMsgType = PIURI + "/ack"
+	// OldPIURI is the old did-exchange protocol identifier URI.
+	OldPIURI = "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/didexchange/1.0"
```

Comment:
@swcurran @andrewwhitehead when does this old PIURI go away?

Reply:
Not sure if this is due to the [msg transition](https://github.com/hyperledger/aries-rfcs/blob/233aeae8f730c8790533e91b52363ed0715ab225/features/0348-transition-msg-type-to-https/README.md), but DID Exchange was one of the "new" protocols. @swcurran @andrewwhitehead 

> Note: Any RFCs that already use the new "https" message type should continue to use the use new format in all cases—accepting and sending. New protocols defined in new and updated RFCs should use the new "https" format.

Reply:
At the moment ACA-Py has a startup option to use the new URI prefix. If the .NET framework now accepts that then we may be able to make it the default in the next release. Changing the prefix is automatic, so it doesn't have custom behaviour depending on the message type (ie. did-exchange).

Reply:
Looks like .NET has HTTPS handling [code](https://github.com/hyperledger/aries-framework-dotnet/blob/074dd0c14ee1bf50613523c55c3c2031a9046a21/src/Hyperledger.Aries/Agents/MessageTypesHttps.cs) for the HTTPs prefixes. Can you please switch the default @swcurran @andrewwhitehead @tmarkovski - we should be using the HTTPS prefix in the test harness.

Reply:
@andrewwhitehead -- ACA-Py should be using the new prefix's on all new protocols anyway -- such as DID Exchange. 

We need to do a quick pass through the various mobile agents to ensure that all still work, but I think we are good with switching.

We really need to figure out how to do these community updates better -- how we can communicate when we can go from step to step reliably.  Basically -- that means we need participation from those involved -- software creators that build the frameworks and those that deploy them.  It's the latter that is the tricky part.